### ISSUE TITLE
Up/down vote not keyboard accessible in inline chat

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#208999]
1. Open inline chat 
2. Ask something, for example "write a sorting function in javascript"
3. Make sure mouse is not over the widget.
4. Move focus to the central part of the widget. The up/down votes are not visible 🐛 

Actually in 4 if I press the `down` key they get visible.
So ideally the up/down vote icons would get visible just by me tabbing through the widget. I would not have to do the extra "down" press.

![Image](https://github.com/microsoft/vscode/assets/1926584/36fab1ab-f8fe-4331-87ca-f8b258b39ccd)

fyi @joaomoreno

**Comment by jrieken** [comment]
Same for quick chat... 

@roblourens can you enlighten us how this works in panel chat?

**Comment by TylerLeonhardt** [comment]
It's technically possible with quick chat:

https://github.com/microsoft/vscode/assets/2644648/4ec03c57-9a80-4417-b04f-d72c9cfa1584

There seems to be an initial tab focus on "the list entirely" which can then be arrowed down and up to show the buttons.

**Comment by TylerLeonhardt** [comment]
My Panel Chat experience is interesting... I _dont_ see the vote buttons until I tab until the "list" is highlighted and use the arrow keys to select a particular row:

https://github.com/microsoft/vscode/assets/2644648/35ee8a3b-372a-4203-b1a0-ec6ac1b122d7

**Comment by jrieken** [comment]
Inline chat now uses a separate toolbar for voting at the bottom which can be tabbed to
